classes/python3-usr.yaml

```diff
@@ -12,6 +13,9 @@ buildSetup: &ref |
             fi
         done
     done
+    for j in "" "lib-dynload" "site-packages" "dist-packages" ; do
+        PYTHONPATH="${PYTHON3_INSTALL_DIR}${PYTHON3_DESTLIB}${j:+/$j}${PYTHONPATH:+:$PYTHONPATH}"
+    done
```

Comment:
Could you move this to the `python3-pkg` class instead?

Reply:
indeed, makes sense.